source/framework/vx_exemplar.c

```diff
@@ -175,7 +175,10 @@ vx_reference tivxCreateReferenceFromExemplar(
         default:
             break;
     }
-
+    if (exemplar->supplementary_data)
```

Comment:
Can you please update the check to be the below for MISRA-C purposes?

if (NULL != exemplar->supplementary_data)

Reply:
done